src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -364,7 +536,7 @@ void Input::cloneBlobIfRequired() {
         prec != element::string &&
         // IRs already have all subnormals flushed to zero, but in
         // read_model scenario with directly loaded original model still can have subnormals
-        isBlobAligned(m_constOp) && (!needFlushDenormalsToZero || !hasSubnormals()) &&
+        isBlobAligned(m_constOp) && (!needFlushDenormalsToZero || !has_subnormals) && !has_bf16_overflows &&
```

Comment:
hasSubnormals() can be saved when (!needFlushDenormalsToZero), you actually removed this optimization here.

Reply:
Added 'needFlushDenormalsToZero' check before 'check_subnormals' and used 'has_subnormals' as the final memory load lable.